ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -540,14 +540,14 @@ keys:
   custom_templates:
     type: list
     display_name: Extensibility with Custom Templates
-    description: "- Custom templates can be added below the playbook directory.\n-
```

Comment:
Hmmm his change should not be happening - it is one of teh reasons the CI is failing.Make sure you are running locally the playbook with libyaml=True

```
user@server # ansible --version
ansible [core 2.12.10]
  config file = ...
  configured module search path = [...]
  ansible python module location = ...
  ansible collection location = ...
  executable location = ...
  python version = 3.10.3 (main, Jul 29 2022, 21:02:56) [Clang 13.0.0 (clang-1300.0.29.30)]
  jinja version = 3.1.2
  libyaml = True